docs/cookbooks-retention-expiry.md

```diff
@@ -37,7 +37,7 @@ Pulsar's [admin interface](admin-api-overview.md) enables you to manage both ret
 By default, when a Pulsar message arrives at a broker, the message is stored until it has been acknowledged on all subscriptions, at which point it is marked for deletion. You can override this behavior and retain messages that have already been acknowledged on all subscriptions by setting a *retention policy* for all topics in a given namespace. Retention is based on both a *size limit* and a *time limit*.
 
 The diagram below illustrates the concept of message retention.
-![Concept of message retention in Pulsar.](/assets/retention.svg)
+![Concept of message retention in Pulsar.](/assets/retention-new.png)
```

Comment:
why are you switching to use .png files? How are these files created? What is the original source of the diagram if someone else would like to edit the diagram?

Reply:
Where are the sources for these diagrams?

Reply:
How did you regenerate the diagram?

Reply:
I recreated it locally using OmniGraffle, and it supports generating SVG format.

Reply:
Would it be possible to share the original OmniGraffle files? It would be useful if anyone could be able to maintain the diagrams that we have in the docs.

Reply:
Of course, but should we upload it directly to this repository? Each file is about 3MB, and I'm worried it might get too large in the future. Can we put it in SVN instead? For example: https://dist.apache.org/repos/dist/release/pulsar/pulsar-site/orignal-diagram

Reply:
I don't think that 3MB is much in repository size. How many files are there for the diagram files?
Let's keep them in pulsar-site git repository, for example in `diagrams` sub directory. That would get ignored in the Docusaurus build so the website size wouldn't increase.

Reply:
that makes sense. I push to ./diagrams directory.